apps/design-system/src/utils/connectors/aws-kms-connector.tsx

```diff
@@ -1,8 +1,18 @@
 import { IFormDefinition } from '@harnessio/forms'
-import { CredTypeValues, DelegateTypes, IInputConfigWithConfigInterface, InputConfigType } from '@harnessio/ui/views'
+import { IInputConfigWithConfigInterface, InputConfigType } from '@harnessio/ui/views'
 
 export const AWS_KMS_CONNECTOR_CATEGORY = 'Secrets Manager'
 
+export enum CredTypeValues {
```

Comment:
is this enum relevant only to AWS KMS connector type?

Reply:
change name to relate only to aws